src/redux/studio-permissions.js

```diff
@@ -1,16 +1,18 @@
-const {selectUserId, selectIsAdmin, selectIsSocial, selectIsLoggedIn, selectUsername} = require('./session');
+const {selectUserId, selectIsAdmin, selectIsSocial,
+    selectIsLoggedIn, selectUsername, selectIsMuted} = require('./session');
 
 // Fine-grain selector helpers - not exported, use the higher level selectors below
 const isCreator = state => selectUserId(state) === state.studio.owner;
 const isCurator = state => state.studio.curator;
 const isManager = state => state.studio.manager || isCreator(state);
 
 // Action-based permissions selectors
-const selectCanEditInfo = state => selectIsAdmin(state) || isManager(state);
+const selectCanEditInfo = state => !selectIsMuted(state) && (selectIsAdmin(state) || isManager(state));
```

Comment:
Just thought this; can ST members get muted (and become unable to edit info)? That would be funny

Reply:
I know paddle2see accidentally banned himself once, so I assume they can be muted too.